docs/book/src/developer/providers/v1.0-to-v1.1.md

```diff
@@ -27,9 +29,11 @@ are kept in sync with the versions used by `sigs.k8s.io/controller-runtime`.
 
 ### API Change
 
-* Some controllers have been moved to internal to reduce there API surface. We now only
+* Some controllers have been moved to internal to reduce their API surface. We now only
   surface what is necessary, e.g. the reconciler and the `SetupWithManager` func:
-    * [bootstrap/kubeadm](https://github.com/kubernetes-sigs/cluster-api/pull/5493) 
+    * [bootstrap/kubeadm/controllers](https://github.com/kubernetes-sigs/cluster-api/pull/5493) 
+    * [exp/addons/controllers](https://github.com/kubernetes-sigs/cluster-api/pull/5639) 
+    * [test/infrastructure/docker/controllers](https://github.com/kubernetes-sigs/cluster-api/pull/5595) 
```

Comment:
FYI, I'm asking contributors to take care of adding those lines as part of their PRs, so every change is auto consistent, doc included.

Reply:
Sounds good. I'll make sure the /controllers suffix is consistently there once the other PRs have been merged (don't want to force another round of nits)